aggregates/src/androidMain/kotlin/org/dhis2/mobile/aggregates/data/DataSetInstanceRepositoryImpl.kt

```diff
@@ -51,4 +61,225 @@ internal class DataSetInstanceRepositoryImpl(
         } ?: DataSetRenderingConfig(
         useVerticalTabs = true,
     )
+
+    override fun categoryOptionCombinations(categoryUids: List<String>): List<String> {
```

Comment:
public methods in repository should be suspend

Reply:
true!